templates/pyramid/files/Pipfile

```diff
@@ -0,0 +1,15 @@
+[[source]]
+url = "https://pypi.org/simple"
+verify_ssl = true
+name = "pypi"
+
+[packages]
+pyramid = "*"
+platformshconfig = "*"
```

Comment:
I don't know if this is typical in Python-land, but I'm a little concerned with this self-updating itself to v3 of the library if we ever release a v3 with a BC break. Is that something we should be concerned about?